erpnext/accounts/doctype/budget/budget.py

```diff
@@ -303,12 +303,10 @@ def get_amount(args, budget):
 	amount = 0
 
 	if args.get("doctype") == "Material Request" and budget.for_material_request:
-		amount = (
-			get_requested_amount(args, budget) + get_ordered_amount(args, budget) + get_actual_expense(args)
```

Comment:
The get_ordered_amount function only gets the unbilled amount so actual expenses still need to be considered. There might be some PO and PI linking issue or the validation may be correct, total unbilled PO and actual expenses might be greater than the allowed budget value.

The validation message can be improved though and all these breakup values can be shown to the user in the validation like

- Total Unbilled order amount (These also can be links to report views for users to verify)
- Actual Expenses
- Material Requests